device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/fileupload/FileUploadTaskTest.java

```diff
@@ -655,89 +660,15 @@ public void runFileUploadResponseParserThrows() throws IOException, IllegalArgum
         Deencapsulation.invoke(fileUploadTask, "run");
     }
 
-    /* Tests_SRS_FILEUPLOADTASK_21_022: [If the upload to blob failed, the run shall create a notification the IoT Hub with `isSuccess` equals false, `statusCode` equals -1.] */
```

Comment:
Can you speak to the removal of this code? Why is it no longer needed?

Reply:
It was testing if the library threw a certain exception, and the new version of the library doesn't throw it anymore